ui/app/styles/bahmni-components/_table.scss

```diff
@@ -224,3 +224,32 @@ table.h-scroll {
 .table-scroll-x {
   overflow-x: scroll;
 }
+
```

Comment:
**Investigation chart** is very specific to **clinical module**. Shouldn't it (this style) be present in **ui/app/styles/clinical** folder?

Reply:
@sumanmaity112  Can you please advise me in which .scss file these CSS changes should go?

Reply:
@sumanmaity112 ok sure. Thank you.